### ISSUE TITLE
`docker pull adoptopenjdk/openjdk11-openj9:latest` gives `no matching manifest for linux/amd64 in the manifest list entries`

### ISSUE THREAD

**Issue by MPV** [adoptium/temurin-build#2719]
**What are you trying to do?**

`docker pull adoptopenjdk/openjdk11-openj9:latest`

**Expected behaviour:**

image pulled

**Observed behaviour:**

```
$ docker pull adoptopenjdk/openjdk11-openj9:latest
latest: Pulling from adoptopenjdk/openjdk11-openj9
no matching manifest for linux/amd64 in the manifest list entries
```

**Any other comments:**

The `latest` image (published 2 hours ago) seems to only be published for the `linux/ppc64le` OS/arch as per:
https://hub.docker.com/layers/adoptopenjdk/openjdk11-openj9/latest/images/sha256-77fc0dd8ea293637ca1631752ffcb80dd9e3928328823c562838200ca4d4240d?context=explore

**Comment by karianna** [comment]
Eclipse Adoptium is no longer providing images for OpenJ9 builds, please see https://developer.ibm.com/languages/java/semeru-runtimes/ for the successor to adoptopenjdk openj9 builds

**Comment by AdamBrousseau** [comment]
Dup eclipse-openj9/openj9#13534
>This is because of the point release. We recently released jdk 11.0.12+7 with OpenJ9 0.27.1 but only on aix and linux on ppc64le. Therefore, the latest release for linux on amd64 and s390x is still with OpenJ9 0.27.0. When the October refresh comes, we should be back to 1 image for all 3 architectures.
